docs/sources/troubleshooting/support-bundles/index.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Learn how to send a support bundle to Grafana Labs support for troubleshooting
+keywords:
+  - grafana
+  - troubleshooting
+  - support
+  - bundles
+title: Send a support bundle to Grafana Labs support
+menutitle: Send a support bundle to support
+weight: 200
+---
+
+# Send a support bundle to Grafana Labs support
```

Comment:
@Jguer , is this available for all users, including OSS? If not, then we need to call out that this feature is for enterprise customers only.

Reply:
Creating a support bundle is available for all users.

Sending these support bundles to GL support is reserved for enterprise

Reply:
Ok, thank you.